src/types/jobs/resolver.js

```diff
@@ -0,0 +1,19 @@
+
+export default {
+  Query: {
+    getJobById: async (_, { id }, { dataSources }) => {
+      console.log(dataSources);
+      return await dataSources.jobsAPI.getJobById(id);
+    }
+  },
+  Job: {
```

Comment:
I don't quite understand why you have `Job` and `RelatedJobs` outside of the `Query` object. I think they both are queries and should go inside the `Query` object

Reply:
This the way to resolve the `related` field in the Job type and `job` in the RelatedJobs type.
Basically, any field can be resolved that's part of the coolest part of Graphql
https://www.apollographql.com/docs/graphql-tools/resolvers.html